### PR TITLE
linux-firmware: add new package r8152-firmware

### PR DESCRIPTION
**Pull request by mj22226**

### CODE REVIEW

File: package/kernel/linux/modules/usb.mk [hunk 1]
```diff
@@ -1340,6 +1340,7 @@ $(eval $(call KernelPackage,usb-net-rtl8150))
 
 define KernelPackage/usb-net-rtl8152
   TITLE:=Kernel module for USB-to-Ethernet Realtek convertors
+  DEPENDS:=+LINUX_5_10:kmod-crypto-sha256 +LINUX_5_10:r8152-firmware
```
Comment (adschm) [inline package/kernel/linux/modules/usb.mk:1343]: Please use +!LINUX_5_4:... instead

Comment (mj22226) [reply package/kernel/linux/modules/usb.mk:1343]: `request_firmware` was added in v5.5 unless is backported or merged in 5.4 will not work.

Comment (adschm) [reply package/kernel/linux/modules/usb.mk:1343]: How that relevant? I just wanted to invert logic so we don't have to touch this again when the next kernel comes. Or will this be removed again after 5.10?

**Comment by adschm** [comment]
Ping. Please fix the issue reported above.

File: package/kernel/linux/modules/usb.mk (hunk 1 above)
Comment (mj22226) [reply package/kernel/linux/modules/usb.mk]: I've done the suggested changes, since this will not be removed after kernel 5.10

**Comment by hauke** [comment]
Thank you for the patch, I applied it to master in 3fe253c6abc2ccc02ea789cffc47761b2381c6ca.

**Pull request closed without merging**